packages/api/src/controllers/userFarmController.js

```diff
@@ -565,6 +565,10 @@ const userFarmController = {
               .patch({
                 email,
                 status_id: 2,
```

Comment:
Can a pseudo user only converted to a worker? I think the frontend should reflect this if that's the case.

Reply:
I think I understood wrong, I believe you can ignore the above comment.